scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -470,6 +474,23 @@ class FossId internal constructor(
                 "The code for an existing scan must not be null."
             }
 
+            log.info { "Loading the ignore rules from '$existingScanCode'." }
+            val ignoreRules = service.listIgnoreRules(config.user, config.apiKey, existingScanCode)
+                .checkResponse("list ignore rules")
+            ignoreRules.data?.let { rules ->
+                log.info { "${rules.size} ignore rule(s) have been found." }
+
+                // When a scan is created with the optional property 'git_repo_url', the server automatically creates
+                // an 'ignore rule' to exclude the '.git' directory.
+                // Therefore, this rule will be created automatically and does not need to be carried from the old scan.
+                rules.filterNot { it.type == RuleType.DIRECTORY && it.value == ".git" }.forEach {
+                    service.createIgnoreRule(config.user, config.apiKey, scanCode, it.type, it.value, RuleScope.SCAN)
+                        .checkResponse("create ignore rules", false)
+                    log.info { "Ignore rule of type '${it.type}' and value '${it.value}' " +
+                            "has been carried to the new scan." }
```

Comment:
Nit: Looks like this line could be wrapped at a later column.

Reply:
No it doesn't. Have I wrong IDE settings ?

![image](https://user-images.githubusercontent.com/66427221/150105389-cbc5e9b5-98f1-4eef-ab77-337b0854c00a.png)

Reply:
By "later" I meant it could be wrapped after the word "new" only. Or be reformatted like

```
log.info {
    "Ignore ..."
}
```